verify-cborld-presentation/src/pages/qr.tsx

```diff
@@ -0,0 +1,27 @@
+const uncompressedQr =
+  "3ECQDJQBQEIRQ4DYEQ2DQMTEGMYTMYZNME4DANBNGQ3TEZRNMI2GEMBNMY4WEOBYG42DEN3DGFSRQ44CU4MHIGDIDB7GIYTMMFUBRAA2MDF77VQYQJ4BSZDFNVXS45TJNEXGIZLWFZWWC5DUOJWGCYTTFZUW6GEGPCIGK6KKNBREOY3JJ5UUURS2IVJFIUKTJFZUS3KJGJHEGSJWLJWUM43DGJKXGSLNJZ4WCWCRNFHWY43JLFVFSMCJNQYTSLROKN3TEUKZGNCGQVDKLA4XAYSYNBDXSVCJJIYWKZDXMZRW6WKNO5MFAMSKMIWXKSCQOJVWU5CLPFJUI52CPBVTEQTHKFIHI5KENVZW6NRUFU2EEOCIOVMTKT2VJBFTC4TPPJXWW4KCKEMIUGEUDCHIGGIEAFMCF3IB2VBLOXOWKI4LE7IIOU726WO7VNUNKSS5UINYL2BIQBRJLLCKVNJFQIXNAHKUFN252ZJDRMT5BB2T7L2Z36VWRVKKLWRBXBPIFCAGFFNMJKVVFJYYOQMGQGD6MRRGYYLIDCABUYGL77LBRATYDFSGK3LPFZ3GS2JOMRSXMLTNMF2HI4TMMFRHGLTJN4MIM6EQMV4UU2DCI5RWST3JJJDFURKSKRIVGSLTJFWUSMSOINETMWTNIZZWGMSVONEW2TTZMFMFC2KPNRZWSWLKLEYES3BRHEXC452RNVRDGLKENJWVQWL2GY3FMNKON4YVASLNNN4FG2SRGJKDQ2KXG43HKUKTGVNC2SKBOE2HETTWKJTGW3ZZORXG6VJNIQ4UKUSWMFREU2TZKVNFKSKTNB5EUSDMJNBWCRLNJRFUIQIYRIMJIGEOQMMQIAKYELWQDDBAQFLNSFSUTDIOJOOHOPJVXVDJOJUFC26OAPCRET4LPY2T2AKSLARO2AMMECAVNWIWKSMNBZFZY5Z5GW6UNFZGQULLZYB4KESPRN7DKPIBKIMHLAIYNYMHRAQZAQAVQIXNAGGCBAKW3ELFJGGQ4S44O46TLPKGS4TIKFV44A6FCJHYW7RVHUAVEGD5QGTQDBARUFTEA5TPMNQWE6BINB2HI4DTHIXS65ZTNFSC433SM4XXGZLDOVZGS5DZF52W4ZDFMZUW4ZLEKRSXE3JDOJUHI5DQOM5C6L3TMNUGK3LBFZXXEZ3YFZUHI5DQOM5C6L3NMF2HI4ROM5WG6YTBNQXWG33OORSXQ5DTF53GGLLFPB2GK3TTNFXW44ZPOYYRQ4VFDB2BQ2AYQANGBS77GUMIM6EQMV4UU2DCI5RWST3JJJDFURKSKRIVGSLTJFWUSMSOINETMWTNIZZWGMSVONEW2TTZMFMFC2KPNRZWSWLKLEYES3BRHEXC4RLXI5VWCV3XNMYGW53UG5CW66S7GUYU6RSXPF2GU6CJJ5IEYZTSGJYG2YLNPEZFEX3XM4ZTE4SKMRUWQYRNNVVVQTSIJJBVUU3COJMDMUTOINYWMNCYOBCVM4CFJRIGEM3YPBXUIUIYRIMJEGEOQMMQIAKYELWQC6NEMDBAHL2G3ZJALQXTEYAKGTZUPL74ULZBDWZT6II5MOAG6RZYLARO2ALZURQMEA5PI3PFEBOC6MTABI2PGR5P7SRPEEO3GPZBDVRYA32HHAMHLAYYNR3FMYLDMNUW4YLUNFXW4Q3FOJ2GSZTJMNQXIZIZCXDBSEKAPAQEGT2WJFCC2MJZEBLGCY3DNFXGC5DJN5XCAQ3FOJ2GSZTJMNQXIZIZCXGKMGDQQIMQIAKYELWQDVKCW5O5MURYWJ6QQ5J7V5M57K3I2VFF3IQ3QXUCRADCSWWEVK2SDEFS46BKGQQEM33SOQQFG5DSMVSXILBAIF2WG23MMFXGIICDIJCCYICBOVRWW3DBNZSCAMJQGEYBSDU6M5IGC43UMV2XEGIO6RSE2YLMMUMQ6FTFJRXWS5LTDEIFAZCDNBSWMGIV2KBBUYGL742BSARDDEK5NIQYOCBBSBABLARO2ALZURQMEA5PI3PFEBOC6MTABI2PGR5P7SRPEEO3GPZBDVRYA32HHAMRCQDHORSXG5DJNZTQ====";
+
+export const getQrData = (): string => compressedQr;
+
+export const QrPage: React.FC = () => {
+  const [qrCode, setQrCode] = useState<string>();
+  useEffect(() => {
+    (async () => {
+      setQrCode(getQrData());
+    })();
+  }, []);
```

Comment:
Not quite understand this block 😬 Are you trying to set the default value for the `qrCode` state?

Reply:
Ah yip good spotting I don't need this block there anymore, was generating QR on the fly before. Will remove